modules/aws/aws-ebs-csi-driver.tf

```diff
@@ -0,0 +1,168 @@
+locals {
+  aws-ebs-csi-driver = merge(
+    local.helm_defaults,
+    {
+      name       = "aws-ebs-csi-driver"
+      namespace  = "kube-system"
+      chart      = "aws-ebs-csi-driver"
+      repository = "https://particuleio.github.io/charts"
+      create_ns  = false
+      service_account_names = {
+        controller = "ebs-csi-controller-sa"
+        snapshot   = "ebs-snapshot-controller"
+      }
+      create_iam_resources_irsa = true
+      create_storage_class      = true
+      storage_class_name        = "ebs-sc"
```

Comment:
`ebs-sc` ? Why not just `ebs` ?

Reply:
because it is the default in upstream repository so it match to not confuse user

Reply:
fair enough